soroban-env-host/src/host/num.rs

```diff
@@ -5,7 +5,7 @@ macro_rules! impl_wrapping_obj_from_num {
             &self,
             _vmcaller: &mut VmCaller<Host>,
             u: $num,
-        ) -> Result<<$hot as HostObjectType>::Wrapper, HostError> {
+        ) -> Result<<$hot as HostObjectType>::Wrapper, Self::Error> {
```

Comment:
```suggestion
        ) -> Result<<$hot as HostObjectType>::Wrapper, HostError> {
```
I _think_ this is accidental edit-churn?

Reply:
Yes, removed.